shell/platform/embedder/embedder.cc

```diff
@@ -132,6 +132,12 @@ static bool IsOpenGLRendererConfigValid(const FlutterRendererConfig* config) {
     return false;
   }
 
+  if (!SAFE_EXISTS(open_gl_config, fbo_with_damage_callback)) {
```

Comment:
update the block above to validate that only one of: `{fbo_callback, fbo_with_frame_info_callback, fbo_with_damage_callback}` is specified.

Reply:
In the current implementation, fbo_with_damage_callback only works if at least fbo_callback or fbo_with_frame_info_callback have been defined. That is because fbo_with_damage_callback takes in an FBO ID argument which is provided by either fbo_callback or fbo_with_frame_info_callback. Therefore, I need to have fbo_with_damage_callback defined and at least one of fbo_callback or fbo_with_frame_info_callback also defined. However, I could also adjust the implementation to avoid the need to pass FBO ID to  fbo_with_damage_callback and have this callback return the ID itself. The benedict of the current implementation is that it adds a callback with a different functionality from the existing ones, which would hopefully reduce the level of confusion around the multiple callbacks.